tests/test_transform_langfilter.py

```diff
@@ -32,7 +32,6 @@ def test_lang_filter(project):
         Kansalliskirjasto on kaikille avoin kulttuuriperintöorganisaatio, joka
         palvelee valtakunnallisesti kansalaisia, tiedeyhteisöjä ja muita
         yhteiskunnan toimijoita.
-        Abc defghij klmnopqr stuwxyz abc defghij klmnopqr stuwxyz.
```

Comment:
I had to change this test. This is a nonsensical sentence that pycld3 is unsure about (`.is_reliable == False`), so the language filter gives it the benefit of the doubt and retains it. Lingua simply identifies it as Swahili, so it gets stripped. 

In the Lingua documentation I can't see a direct way of telling whether Lingua is unsure about some input; there is the [minimum relative distance](https://github.com/pemistahl/lingua-py#92-minimum-relative-distance) parameter which could be tuned, but I'm not sure that it would help with nonsensical input. For example, when converting PDF documents to text, it's quite likely to result in garbage "sentences" that aren't in any real language.